indent/python.vim

```diff
@@ -421,3 +424,123 @@ function! GetPythonPEPIndent(lnum)
 
     return s:indent_like_previous_line(a:lnum)
 endfunction
+
+function s:SearchPosWithSkip(pattern, flags, skip, stopline)
+    "
+    " Returns true if a match is found for {pattern}, but ignores matches
+    " where {skip} evaluates to false. This allows you to do nifty things
+    " like, say, only matching outside comments, only on odd-numbered lines,
+    " or whatever else you like.
+    "
+    " Mimics the built-in search() function, but adds a {skip} expression
+    " like that available in searchpair() and searchpairpos().
+    " (See the Vim help on search() for details of the other parameters.)
+    "
+    " Note the current position, so that if there are no unskipped
+    " matches, the cursor can be restored to this location.
+    "
+    let l:flags = a:flags
+    let l:movepos = getpos('.')
+    let l:firstmatch = []
+    let l:pos = [0, 0, 0, 0]
+
+    " Loop as long as {pattern} continues to be found.
+    "
+    while search(a:pattern, l:flags, a:stopline) > 0
+        if l:firstmatch == []
+            let l:firstmatch = getpos('.')
+            let l:flags = substitute(l:flags, 'c', '', '')
+        elseif l:firstmatch == getpos('.')
+            break
+        endif
+
+        " If {skip} is true, ignore this match and continue searching.
+        "
+        if eval(a:skip)
+            continue
+        endif
+
+        " If we get here, {pattern} was found and {skip} is false,
+        " so this is a match we don't want to ignore. Update the
+        " match position and stop searching.
+        "
+        let l:pos = getpos('.')
+        let l:movepos = getpos('.')
+        break
+
+    endwhile
+
+    " Jump to the position of the unskipped match, or to the original
+    " position if there wasn't one.
+    "
+
+    call setpos('.', l:movepos)
+    return [l:pos[1], l:pos[2]]
+
+endfunction
+
+function s:IsInComment(lnum, col)
+    echom synIDattr(synID(a:lnum, a:col, 1), 'name')
+    return synIDattr(synID(a:lnum, a:col, 1), 'name') =~? 'comment'
+endfunction
+
+function! GetPythonPEPFormat(lnum, count)
+  let l:tw = &textwidth ? &textwidth : 79
```

Comment:
The maximum line length according to PEP8 is 79. https://www.python.org/dev/peps/pep-0008/#maximum-line-length